config.coffee

```diff
@@ -68,8 +68,9 @@ module.exports =
   CRITEO_AUCTIONS_ACCOUNT_NUMBER: 28539
   CRITEO_ARTWORKS_ACCOUNT_NUMBER: 35250
   MARKETING_SIGNUP_MODAL_COPY: 'Sign up for early access'
-  MARKETING_SIGNUP_MODAL_IMG: 'http://placekitten.com/200/200'
+  MARKETING_SIGNUP_MODAL_IMG: 'http://lorempixel.com/200/200/sports/'
```

Comment:
I added an alternative image placeholder because the other seems to be a broken link for the `200/200` image.